torch_geometric/transforms/add_metapaths.py

```diff
@@ -192,3 +193,183 @@ def _get_edge_weight(self, data: HeteroData,
         else:
             edge_weight = None
         return edge_weight
+
+
+@functional_transform('add_metapaths2')
+class AddMetaPaths2(BaseTransform):
```

Comment:
Lets call this function `AddRandomMetaPaths`. I feel like its functionality is different enough, to not merge it with `AddMetaPaths` and this name seems to better indicate how it works.

Reply:
Got it. Will update the code soon.